platform-ui-skin/src/main/webapp/skin/less/social/skin/SocialPage/Style.less

```diff
@@ -72,7 +72,24 @@
 #singlePageApplicationContainer, .singlePageApplication, .SpaceActivityStreamPortletPage {
   max-width: 100%;
   width: @maxTabletWidth;
-  padding: 20px 14%;
+  margin: 22px auto 10px;
+}
+
+.SpaceActivityStreamPortletPage {
```

Comment:
Could you change the selector above instead of duplicating the same rules

Reply:
It was a useless part that I should remove

Reply:
fixed @ahamdi